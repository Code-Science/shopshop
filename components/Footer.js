import React from 'react';
import Link from 'next/link';
import FacebookIcon from '@material-ui/icons/Facebook';
import TwitterIcon from '@material-ui/icons/Twitter';
import InstagramIcon from '@material-ui/icons/Instagram';
import styles from '../styles/components/Footer.module.css';

function Footer() {
  return (
    <>
      <div className={styles.footer}>
        <div className={styles.section}>
          <p className={styles.footerHeading}>Customer Care</p>
          <ul>
            <li>
              <Link href="/">
                <a className={styles.navigation}>CONTACT US</a>
              </Link>
            </li>
            <li>
              <Link href="/">
                <a className={styles.navigation}>STORE LOCATION</a>
              </Link>
            </li>
            <li>
              <Link href="/">
                <a className={styles.navigation}>CAREERS</a>
              </Link>
            </li>
            <li>
              <Link href="/">
                <a className={styles.navigation}>TERMS OF USE</a>
              </Link>
            </li>
            <li>
              <Link href="/">
                <a className={styles.navigation}>RETURNS</a>
              </Link>
            </li>
            <li>
              <Link href="/">
                <a className={styles.navigation}>PRIVACY POLICY</a>
              </Link>
            </li>
          </ul>
        </div>
        <div className={styles.section}>
          <p className={styles.footerHeading}>About</p>
          <div className={styles.about}>
            <p className={styles.name}>Shopshop</p>
            <p>Curators of ethical fashion and New shopshop designed.</p>
            <p>
              Shopshop is a destination for style seekers. Located in xyzton's
              boutique xyz Village, shopshop is lovingly and carefully lorem
              lorem lorem . Dedicated to stylish designed, ethical fashion and
              accessories- shopshop lorem lorem is a beautiful style edit.
            </p>
          </div>
        </div>
        <div className={styles.section}>
          <p className={styles.footerHeading}>Newsletter</p>
          <p className={styles.newsletter}>Join our mailing list</p>
          <form className={styles.form}>
            <input
              type="text"
              placeholder="your@email.com"
              className={styles.input}
            />
            <button type="button" className={styles.subscribeBtn}>
              Subscribe
            </button>
          </form>
        </div>
      </div>
      <div className={styles.bottom}>
        <div className={styles.icons}>
          <FacebookIcon style={{ fontSize: 30 }} />
          <TwitterIcon style={{ fontSize: 30 }} />
          <InstagramIcon style={{ fontSize: 30 }} />
        </div>
        <div className={styles.copyright}>Copyright Â© Shopshop 2020</div>
      </div>
    </>
  );
}

export default Footer;
