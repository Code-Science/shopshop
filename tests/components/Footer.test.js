import { configure, shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import React from 'react';
import Footer from '../../components/Footer';

jest.mock('../../store/StateProvider');

configure({ adapter: new Adapter() });

describe('Footer Component', () => {
  const wrapper = shallow(<Footer />);

  it('expect to render Footer component', () => {
    expect(wrapper.length).toEqual(1);
  });
  it('snapshot test', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
