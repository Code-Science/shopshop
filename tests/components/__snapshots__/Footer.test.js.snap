// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Component snapshot test 1`] = `
<Fragment>
  <div
    className="footer"
  >
    <div
      className="section"
    >
      <p
        className="footerHeading"
      >
        Customer Care
      </p>
      <ul>
        <li>
          <Link
            href="/"
          >
            <a
              className="navigation"
            >
              CONTACT US
            </a>
          </Link>
        </li>
        <li>
          <Link
            href="/"
          >
            <a
              className="navigation"
            >
              STORE LOCATION
            </a>
          </Link>
        </li>
        <li>
          <Link
            href="/"
          >
            <a
              className="navigation"
            >
              CAREERS
            </a>
          </Link>
        </li>
        <li>
          <Link
            href="/"
          >
            <a
              className="navigation"
            >
              TERMS OF USE
            </a>
          </Link>
        </li>
        <li>
          <Link
            href="/"
          >
            <a
              className="navigation"
            >
              RETURNS
            </a>
          </Link>
        </li>
        <li>
          <Link
            href="/"
          >
            <a
              className="navigation"
            >
              PRIVACY POLICY
            </a>
          </Link>
        </li>
      </ul>
    </div>
    <div
      className="section"
    >
      <p
        className="footerHeading"
      >
        About
      </p>
      <div
        className="about"
      >
        <p
          className="name"
        >
          Shopshop
        </p>
        <p>
          Curators of ethical fashion and New shopshop designed.
        </p>
        <p>
          Shopshop is a destination for style seekers. Located in xyzton's boutique xyz Village, shopshop is lovingly and carefully lorem lorem lorem . Dedicated to stylish designed, ethical fashion and accessories- shopshop lorem lorem is a beautiful style edit.
        </p>
      </div>
    </div>
    <div
      className="section"
    >
      <p
        className="footerHeading"
      >
        Newsletter
      </p>
      <p
        className="newsletter"
      >
        Join our mailing list
      </p>
      <form
        className="form"
      >
        <input
          className="input"
          placeholder="your@email.com"
          type="text"
        />
        <button
          className="subscribeBtn"
          type="button"
        >
          Subscribe
        </button>
      </form>
    </div>
  </div>
  <div
    className="bottom"
  >
    <div
      className="icons"
    >
      <FacebookIcon
        style={
          Object {
            "fontSize": 30,
          }
        }
      />
      <TwitterIcon
        style={
          Object {
            "fontSize": 30,
          }
        }
      />
      <InstagramIcon
        style={
          Object {
            "fontSize": 30,
          }
        }
      />
    </div>
    <div
      className="copyright"
    >
      Copyright Â© Shopshop 2020
    </div>
  </div>
</Fragment>
`;
